scripts/cargo-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Run various invocations of cargo check
+
```

Comment:
nit: I'd add a `set -e` to make the whole script fail as soon as there's an error

Reply:
Nice, cheers!